src/styles/block/calendar.scss

```
@@ -0,0 +1,29 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+  width: getSize($cell-size, $cell-gap);
+
+  &__day {
+    @include square($cell-size);
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+
+    border: 1px solid #000;
```

don't forget to create variables for all colors
```suggestion
    border: 1px solid $c-black;
```